examples/bert/bert_auto_opt_gpu.json

```diff
@@ -50,7 +66,8 @@
             }
         },
         "evaluator": "common_evaluator",
-        "execution_providers": ["CUDAExecutionProvider", "TensorrtExecutionProvider"],
```

Comment:
I think, we might have to print a warning if the config class sees this option. Since a user might use an old config and it would run without error.

Reply:
The no error run makes the config appear backward compatible but it is not.

Reply:
we should raise ValidationError if user specify the execution provider by legacy way.

Reply:
updated.